src/geoserver/util.py

```diff
@@ -1055,3 +1057,8 @@ def get_epsg_code(crs):
     authority, epsg_code = crs.split(':')
     assert authority == 'EPSG'
     return int(epsg_code)
+
+
+def get_gs_bbox(crs, bbox,):
```

Comment:
Lze mít tuto metodu jako vlastnost crs_def, buď odkaz nebo lambda funkci.

Reply:
get_gs_bbox -> crs.util.get_wms_bbox(crs, bbox, wms_version)

Introduce boolean key `axes_order_east_north_in_epsg_db` in crs_def. It's False for 4326, True for 3857.